code/modules/projectiles/sentries.dm

```diff
@@ -458,3 +458,46 @@
 	. = ..()
 	var/obj/item/weapon/gun/internal_gun = internal_item
 	. += image('icons/Marine/sentry.dmi', src, internal_gun.placed_overlay_iconstate, dir = dir)
+
+
+//Throwable turret
+/obj/machinery/deployable/mounted/sentry/cope
+	density = FALSE
+
+
+/obj/machinery/deployable/mounted/sentry/cope/sentry_start_fire()
+	var/obj/item/weapon/gun/internal_gun = internal_item
+	internal_gun.update_ammo_count() //checks if the battery has recharged enough to fire
+	return ..()
+
+///Dissassembles the device
+/obj/machinery/deployable/mounted/sentry/cope/disassemble(mob/user)
+	var/obj/item/item = internal_item
+	if(CHECK_BITFIELD(item.flags_item, DEPLOYED_NO_PICKUP))
+		to_chat(user, span_notice("The [src] is anchored in place and cannot be disassembled."))
```

Comment:
Dont use the because byond auto does proper and improper nouns (ie The the thing is anchored)